### PR TITLE
Fix - HEAL overwrites the transcription status for successfully encoded videos

### PR DESCRIPTION
**Pull request by Qubad786**
[EDUCATOR-1431](https://openedx.atlassian.net/browse/EDUCATOR-1431)
- Adds `source_language` and `transcript_status` to VideoSerializer and Admin.
- Fix - HEAL changing transcript statuses for the completed Videos in edx-val.
- Adds test coverage for Serializers under `VEDA_OS01`.

### CODE REVIEW

**Comment by Qubad786** [comment]
@muzaffaryousaf added both source language and transcript status.

**Comment by Qubad786** [comment]
@muhammad-ammar the build is passing locally and on feature branch.

**Comment by Qubad786** [comment]
@yro please have a look at 2nd commit.

**Comment by Qubad786** [comment]
Build is fixed.

**Comment by Qubad786** [comment]
@muzaffaryousaf ^^

screen shot 2017-10-09 at 8 29 45 pm

**Comment by muzaffaryousaf** [comment]
@Qubad786 :1st_place_medal:

File: control/veda_heal.py [hunk 1]
```diff
@@ -73,6 +78,11 @@ def send_encodes(self):
             encode_list = self.determine_fault(video_object=v)
             # Using the 'Video Proto' Model
             if self.val_status is not None:
+                # Update to VAL is also happening for those videos which are already marked complete,
+                # All these retries are for the data-parity between VAL and VEDA, as calls to VAL api are
+                # unreliable and times out. For a completed Video, VEDA heal will keep doing this unless
+                # the Video is old enough and escapes from the time-span that HEAL is picking up on.
+                # cc Greg Martin
                 VAC = VALAPICall(
```
Comment (yro) [inline control/veda_heal.py]: 👍

**Pull request merged**